components/script/dom/node.rs

```diff
@@ -289,7 +287,6 @@ impl Node {
         }
 
         self.owner_doc().content_and_heritage_changed(self, NodeDamage::OtherNodeDamage);
-        child.owner_doc().content_and_heritage_changed(child, NodeDamage::OtherNodeDamage);
```

Comment:
The child needs the version counter changed here too, probably splitting both concepts would be ideal.